receiver/gitproviderreceiver/metadata.yaml

```diff
@@ -42,21 +42,56 @@ metrics:
   git.repository.count:
     enabled: true
     description: Number of repositories in an organization
-    unit: "{repository}"
+    unit: '{repository}'
     gauge:
       value_type: int
     attributes: []
   git.repository.branch.count:
     enabled: true
     description: Number of branches in a repository
-    unit: "{branch}"
+    unit: '{branch}'
     gauge:
       value_type: int
     attributes: [repository.name]
+  git.repository.branch.time:
+    enabled: true
+    description: Time the branch has existed.
+    unit: s
+    gauge:
+      value_type: int
+    attributes: [repository.name, branch.name]
+  git.repository.branch.commit.aheadby_count:
+    enabled: true
+    description: Number of commits a branch is ahead of the default branch.
+    unit: '{commit}'
+    gauge:
+      value_type: int
+    attributes: [repository.name, branch.name]
+  git.repository.branch.commit.behindby_count:
+    enabled: true
+    description: Number of commits a branch is behing the default branch.
+    unit: '{commit}'
+    gauge:
+      value_type: int
+    attributes: [repository.name, branch.name]
+  git.repository.branch.line.addition_count:
+    enabled: true
+    description: Count of lines added to code in a branch
```

Comment:
Should we also add in the description of the `git.repository.branch.line` that these values are relative to the default branch?

Reply:
Yea, I like that idea! I'll adjust the decsription.